Dockerfile

```diff
@@ -0,0 +1,19 @@
+# Pull base image.
+FROM ubuntu:14.04
+
+WORKDIR /mozilla-central
+
+RUN sudo apt-get -y update && \
+    sudo apt-get -y install mercurial wget &&  \
+    apt-get -y --purge dist-upgrade && \
+    apt-get -y --purge autoremove -qq && \
+    apt-get clean -y
+
+# update to latest nightly Firefox
+RUN rm -rf firefox-nightly && \
+    wget -O firefox-nightly.tar.bz2 'https://download.mozilla.org/?product=firefox-nightly-latest-ssl&os=linux64&lang=en-US' && \
```

Comment:
Should this use **ff-tool** or [**mozdownload**](https://github.com/mozilla/mozdownload) directly?

Reply:
@pdehaan i think in general simplifying things using a downloader like mozdownload is wise except in the context of docker, where keeping the image size as small as possible is high priority.  wget'ing a tar.bz2 file seems pretty quick and not too dirty (doesn't require python to accomplish).